nestle.py

```diff
@@ -856,16 +947,23 @@ def prior_transform(u):
         if update_interval < 1:
             raise ValueError("update_interval must be >= 1")
 
+    # Parallel evaluation.
+    if n_threads is None or n_threads == 1:
+        n_threads = 1
+        pool = FakePool()
+    else:
+        if pool is None:
+            pool = concurrent.futures.ThreadPoolExecutor(n_threads)
```

Comment:
I recommend throwing an error to avoid a stream of bug reports saying "parallelization doesn't make it any faster!"